packages/ckeditor5-image/src/image/converters.js

```diff
@@ -69,6 +69,7 @@ export function viewFigureToModel() {
 export function srcsetAttributeConverter() {
 	return dispatcher => {
 		dispatcher.on( 'attribute:srcset:image', converter );
+		dispatcher.on( 'attribute:srcset:imageInline', converter );
```

Comment:
Those should be registered separately by `ImageBlock` and `ImageInline`.

Reply:
Done.